distributions/openhab/src/main/resources/bin/backup

```diff
@@ -12,14 +12,18 @@ getFullPath() {
 
 setup(){
   if [ "$1" = "--help" ] || [ "$1" = "-h" ]; then
-    echo "Usage: backup [filename]"
+    echo "Usage: backup [--full] [filename]"
     echo ""
-    echo "  e.g. ./backup                 << Makes a file with a timed filename"
-    echo "       ./backup myBackup.zip    << Makes a file called myBackup.zip"
+    echo "  e.g. ./backup                     << Makes a file with a timed filename"
+    echo "       ./backup myBackup.zip        << Makes a file called myBackup.zip"
+    echo "  e.g. ./backup --full              << Makes a full backup file with a timed filename"
```

Comment:
Formatting comment: need for the 2nd "e.g." here.

Reply:
Yep, well spotted :-)